server/mount-glustervol

```diff
@@ -18,15 +18,33 @@ with open(info_file) as f:
 
```

Comment:
Question: Where is this file actually being used in the code, I missed this from the get go?
I can observe similar code in at-least two other places (in generating_client_volfile and in shd) and I believe handler methods and above might need some refactoring :/

Reply:
Nope, this is not used anywhere IMO.